config/kernel-signal-stop.m4

```diff
@@ -0,0 +1,21 @@
+dnl #
+dnl # 4.4 API change
+dnl # Added kernel_signal_stop
+dnl #
+AC_DEFUN([ZFS_AC_KERNEL_SRC_SIGNAL_STOP], [
+	ZFS_LINUX_TEST_SRC([signal_stop], [
+		#include <linux/sched/signal.h>
+	],[
+		kernel_signal_stop();
+	])
+])
+
+AC_DEFUN([ZFS_AC_KERNEL_SIGNAL_STOP], [
+	AC_MSG_CHECKING([whether signal_stop() API exists])
```

Comment:
nit: for better or worse most of the other checks format the message as "whether <function> exists".  Let's be consistent.

```suggestion
	AC_MSG_CHECKING([whether signal_stop() exists])
```